code/modules/antagonists/blob/blob/blobs/shield.dm

```diff
@@ -20,14 +21,37 @@
 
 /obj/structure/blob/shield/update_icon()
 	..()
-	if(obj_integrity <= 75)
-		icon_state = "blob_shield_damaged"
-		name = "weakened strong blob"
-		desc = "A wall of twitching tendrils."
+	if(obj_integrity < max_integrity * 0.5)
+		icon_state = "[initial(icon_state)]_damaged"
+		name = "weakened [initial(name)]"
+		desc = "[damaged_desc]"
 		atmosblock = FALSE
 	else
 		icon_state = initial(icon_state)
 		name = initial(name)
 		desc = initial(desc)
 		atmosblock = TRUE
 	air_update_turf(1)
+
+/obj/structure/blob/shield/reflective
+	name = "reflective blob"
+	desc = "A solid wall of slightly twitching tendrils with a reflective glow."
+	damaged_desc = "A wall of twitching tendrils with a reflective glow."
+	icon_state = "blob_glow"
+	flags_1 = CHECK_RICOCHET_1
+	point_return = 8
+	max_integrity = 50
+	brute_resist = 1

```

Comment:
note that reducing the integrity and brute resistance both means that its eHPvsBrute compared to a normal shield blob is 50, vs 600. Is that how weak these should be versus brute?

Reply:
Lasers have an innate 80% chance to reflect, so this is meant to be a good counter against people spamming lasers at you and nothing else. My ideal scenario is blob places these down and the crew now has to either switch to ballistics to break through or rush up and break it themselves.